lexers/sed.go

```diff
@@ -0,0 +1,35 @@
+package s
```

Comment:
Was this converted from Pygments? If so can you please use `_tools/pygments2chroma_xml.py` instead, I'm preferring XML lexers from now on.